packages/coverage-v8/src/provider.ts

```diff
@@ -283,6 +283,19 @@ export class V8CoverageProvider extends BaseCoverageProvider<ResolvedCoverageOpt
           ) {
             return true
           }
+
+          // in-source test with "if (import.meta.vitest)"
+          if (
+            (type === 'branch' || type === 'statement')
+            && node.type === 'IfStatement'
+            && node.test.type === 'MemberExpression'
+            && node.test.object.type === 'Identifier'
+            && node.test.object.name === '__vite_ssr_import_meta__'
```

Comment:
Would also be nice to support `import.meta.vitest` directly (the browser mode doesn't transform it into `__vite_ssr_import_meta__`)

Reply:
Good point! We should enable AST-aware remapping for browser mode in `test/coverage-test` to catch this. So far it's only been enabled for Node.